pkg/util/iptables/iptables.go

```diff
@@ -503,6 +518,18 @@ func getIPTablesWaitFlag(vstring string) []string {
 	}
 }
 
+// Checks if iptables-restore has a "wait" flag
+func getIPTablesRestoreWaitFlag(exec utilexec.Interface) restoreWaitFlag {
+	glog.V(4).Infof("checking if iptables-restore can wait")
+	bytes, err := exec.Command(cmdIPTablesRestore, "--help").CombinedOutput()
+	if err != nil {
+		return restoreWaitUnknown
+	} else if strings.Contains(string(bytes), "[-w secs]") {
```

Comment:
@danwinship done; submitted a patch to netfilter-devel@ to add --version/-V argument; if that gets accepted I will update this PR to check for version 1.6.2 and later.  Still doesn't help us for backported wait patches (like in RHEL) though, thoughts on that?  Would we just have to patch locally for OpenShift or should we still keep the help text checking here?